composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "soberwp/intervention",
+  "name": "runofthemill/intervention",
```

Comment:
☝️ I’m assuming this wasn’t meant for the PR.

Reply:
ah, that was so I could use it with composer - forgot subsequent commits would affect the PR =D

I'll fix now :)

Reply:
@knowler fixed!